NebulaModel/Config.cs

```diff
@@ -1,67 +1,77 @@
 ﻿using BepInEx;
+using BepInEx.Configuration;
+using HarmonyLib;
 using NebulaModel.Logger;
 using System;
+using System.Collections.Generic;
 using System.IO;
-using System.Runtime.Serialization.Formatters.Binary;
+using System.Linq;
+using System.Reflection;
 
 namespace NebulaModel
 {
     public static class Config
     {
         private const string OPTION_SAVE_FILE = "nebula.cfg";
+        private const string SECTION_NAME = "Nebula - Settings";
 
         public static PluginInfo ModInfo { get; set; }
         public static string ModVersion => ThisAssembly.AssemblyInformationalVersion;
         public static MultiplayerOptions Options { get; set; }
 
+        private static ConfigFile configFile;
+
         public static bool LoadOptions()
         {
             Options = new MultiplayerOptions();
 
-            string path = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments), GameConfig.gameName, OPTION_SAVE_FILE);
-            if (File.Exists(path))
+            try
             {
-                try
-                {
-                    var formatter = new BinaryFormatter();
-                    using (Stream stream = new FileStream(path, FileMode.Open, FileAccess.Read, FileShare.None))
-                    {
-                        Options = (MultiplayerOptions)formatter.Deserialize(stream);
-                    }
-                }
-                catch(System.Exception e)
+                configFile = new ConfigFile(Path.Combine(Paths.ConfigPath, OPTION_SAVE_FILE), true);
```

Comment:
We don't really need to specify the path, it's probably best to just use the default config location? (`BepInEx\config`)